seisbench/models/base.py

```diff
@@ -1519,273 +1478,363 @@ def _cut_fragments_point(self, t0, block, stations, argdict):
         :return:
         """
         stride = self._argdict_get_with_default(argdict, "stride")
-        starts = np.arange(0, block.shape[-1] - self.in_samples + 1, stride)
-        if len(starts) == 0:
+        offsets = np.arange(0, group.data.shape[-1] - self.in_samples + 1, stride)
+        in_samples, pred_sample = self._get_in_pred_samples(group)
+
+        if len(offsets) == 0:
             seisbench.logger.warning(
                 "Parts of the input stream consist of fragments shorter than the number "
                 "of input samples. Output might be empty."
             )
-            return
+            return []
 
-        bucket_id = np.random.randint(1000000)
+        bucket_id = np.random.randint(1e9)
```

Comment:
This `bucket_id` is never used, is it? (It also sounds like a terrible idea on my side if we ever used random numbers as hopefully unique identifiers.) If it's unused, we could remove it from the code.

Reply:
removed it.